server/controllers/bookingsController.ts

```diff
@@ -30,4 +32,17 @@ export default class BookingsController {
       return res.redirect(`/premises/${premisesId}/bookings/${confirmedBooking.id}/confirmation`)
     }
   }
+
+  confirm(): RequestHandler {
+    return async (req: Request, res: Response) => {
+      const { premisesId, bookingId } = req.params
+      const booking = await this.bookingService.getBooking(premisesId, bookingId)
+
+      return res.render('premises/bookings/confirm', {
+        ...booking,
+        arrivalDate: parseISO(booking.arrivalDate).toLocaleDateString('en-GB'),
+        expectedDepartureDate: parseISO(booking.expectedDepartureDate).toLocaleDateString('en-GB'),
```

Comment:
I think I'd prefer to see this conversion in the service, in the interests of keeping our controllers as slim as possible 👍

Reply:
Yep will do this